### PR TITLE
Add install info to readme

### PR DESCRIPTION
**Pull request by mlvandijk**
Add info on how to install pip to install Pygments, and how to run HTMLProofer locally.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,19 @@ be highlighted.
 
     pip install Pygments
 
+If you don't have `pip` installed yet, you'll first need to install it.
+
+On Mac OS you can do so by installing Python with homebrew
```
Comment (olleolleolle) [inline README.md]: In before someone else notes the brand guidelines:

- `macOS`
- `Homebrew`

Also: This is a great change. Thanks for keeping the lights on!

File: README.md [hunk 2]
```diff
@@ -68,6 +68,19 @@ be highlighted.
...
+
+    brew install python
+
+We use HTMLProofer to check whether hyperlinks are correct. To run HTMLProofer on your local version, run `make htmlproofer` in the directory
+where this project is located on your machine.
+
+If this doesn't work, you may need to install Ruby first:
```
Comment (olleolleolle) [inline README.md:80]: Perhaps: "If the following does not work"

**Review by olleolleolle — COMMENTED** [review]
Made some inline comments.

File: README.md (hunk 2 above)
Comment (mlvandijk) [reply README.md:80]: "this" actually refers to `make htmlproofer` (I didn't have ruby yet...)

**Review by olleolleolle — APPROVED** [review]

**Pull request merged**